src/common/components/play/multiplayer/LobbyStatus.js

```diff
@@ -22,11 +22,13 @@ const LobbyStatus = (props) => {
   return (
     <Paper style={{padding: 20, marginBottom: 20}}>
       <div style={{position: 'relative'}}>
-        { connected ? <span>
+        { connected ? <span className="players-online">
```

Comment:
Something like:

```
const styles = {
  playersOnline: {
    'span:after': {
      content: ','
    }
  }
};
```

and then

```
<span className={classes.playersOnline} />
```

and

```
export default withStyles(styles)(LobbyStatus);
```

would be more in line with MUI v1.x

Reply:
What about the `.players-online span:last-of-type::after` CSS rule? Could I make that work as well using MUI v1.x styling?

Reply:
Yup same idea